draft-ietf-scitt-architecture.md

```diff
@@ -188,10 +188,10 @@ Auditor:
 
 : an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
 
-Client:
+Relying Party:
 
-: an application making protected Transparency Service resource requests on behalf of the resource owner and with its authorization.
-The term "Client" does not imply any particular implementation characteristics (e.g., whether the application executes on a server, a desktop, or other devices).
+: a client application for protected Transparency Service resource requests on behalf of the resource owner and with its authorization.
```

Comment:
```suggestion
: a relying party depends on or is a Verifier of Signed or Transparent Statements.
```

Reply:
Personally, I'd like to keep CLient as defined and add Relying Party. I'd decline this suggestion, here.